client/components/rewind-credentials-form/index.jsx

```diff
@@ -154,7 +154,7 @@ export class RewindCredentialsForm extends Component {
 				<QueryRewindState siteId={ siteId } />
 				<div className="rewind-credentials-form__instructions">
 					{ translate(
-						'Please get your credentials from your hosting provider. Their website should explain how to get or create the credentials you need. {{link}}Check out our handy guide{{/link}}',
+						'Your server credentials can be found with your hosting provider. Their website should explain how to get the credentials you need. {{link}}Check out our handy guide for more info{{/link}}.',
```

Comment:
This doesn't sound like correct English ... "Your server credentials can be found with your hosting provider" should be something like "You can **_obtain from_** your hosting provider"